kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportConfig.kt

```diff
@@ -627,41 +803,36 @@ public interface KoverHtmlReportConfig {
  *  ...
  * ```
  */
-public interface KoverXmlReportConfig {
+public interface KoverXmlTaskConfig {
     /**
      * Override common filters only for XML report.
      */
-    public fun filters(config: Action<KoverReportFilters>)
+    @Deprecated(
+        message = "",
+        level = DeprecationLevel.ERROR
+    )
+    public fun filters(config: Action<KoverReportFiltersConfig>)
```

Comment:
Yes. I have not met more than one real use-case where it would be useful, for example, to have different filters in an XML and HTML report.

This is very confusing, it is difficult for users to understand exactly where to write `filters` - which leads to the fact of copy-paste the same filters are specified for all types of reports (inside `xml`, `html`, `verify` blocks).

Since it is now possible to create custom variants of reports, if someone needs to generate a report with a different set of filters, then for this user can create a new variant and override the filters for it.